enterprise/server/ociregistry/ociregistry.go

```diff
@@ -460,6 +471,18 @@ func fetchBlobOrManifestFromCache(ctx context.Context, w http.ResponseWriter, bs
 }
 
 func writeBlobOrManifestToCacheAndResponse(ctx context.Context, upstream io.Reader, w io.Writer, bsClient bspb.ByteStreamClient, acClient repb.ActionCacheClient, ref gcrname.Reference, ociResourceType ocipb.OCIResourceType, hash gcr.Hash, contentType string, contentLength int64) error {
+	r := upstream
+	if ociResourceType == ocipb.OCIResourceType_MANIFEST {
+		raw, err := io.ReadAll(upstream)
+		if err != nil {
+			return err
+		}
+		if err := writeManifestToAC(ctx, raw, acClient, ref, hash, contentType); err != nil {
+			log.CtxWarningf(ctx, "error writing manifest to AC: %s", err)
```

Comment:
```suggestion
			log.CtxWarningf(ctx, "Error writing manifest to AC: %s", err)
```

Reply:
I'll clean these up (along with the %q) in a separate change. Thanks for the guidance!